src/main/java/org/smartregister/pathevaluator/dao/QueuingHelper.java

```diff
@@ -0,0 +1,9 @@
+package org.smartregister.pathevaluator.dao;
+
+import org.smartregister.pathevaluator.TriggerType;
+
+public interface QueuingHelper {
```

Comment:
Usually helper classes are utils. Has this been discussed with team. Do we want Service and ServiceImpl instead?